backend/experiment/actions/final.py

```diff
@@ -160,13 +162,19 @@ def action(self) -> FinalActionResponse:
             "logo": self.logo,
         }
 
-        if self.percentile is None:
-            response.pop("rank")
-
         return response
 
     def get_social_media_config(self, session: Session) -> Optional[SocialMediaConfigConfiguration]:
         experiment = session.block.phase.experiment
         if hasattr(experiment, "social_media_config") and experiment.social_media_config:
             return serialize_social_media_config(experiment.social_media_config, session.total_score())
         return None
+
+    def wrap_plain_final_text(self):
+        '''check if `final_text` starts with a html tag
+        If not, wrap it in a `<center>` element for better alignment
+        '''
+        tag_pattern = re.compile('<[a-z]*>')
+        if not re.match(tag_pattern, self.final_text):
+            return f'<center>{self.final_text}</center>'
+        return self.final_text
```

Comment:
@drikusroor do you see any adders under the grass with this solution?

Reply:
Not immediately, but perhaps some test scenarios could be useful here?